### PR TITLE
Clean-up the actions and the properties page for the product binding

### PR DESCRIPTION
**Pull request by laurentiumagureanu**
## Description

Clean-up the actions and the properties page for the product binding

## Motivation and Context
Some actions for the product binding don't make sense anymore, now that the product binding is created automatically.
The "create", "move", "publish", "unpublish" and "delete" must be removed

The "properties" action can stay, but the properties page must be updated to contain the following fields:

Binding title - read only
Configuration reference - a link to the Context Configuration that manages this binding. Ideally this link can open the actual configuration for editing.
## Screenshots (if appropriate):
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

File: content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<%--
```
Comment (dplaton) [inline content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp]: Why is this a JSP? Is this somethin we overlay from a parent component?

File: content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/_cq_dialog/.content.xml [hunk 2]
```diff
@@ -36,29 +36,18 @@
...
-                                            jcr:primaryType="nt:unstructured"
-                                            sling:resourceType="granite/ui/components/foundation/form/textfield"
-                                            cq-msm-lockable="cq:catalogDataResourceProviderFactory"
-                                            disabled="{Boolean}true"
-                                            fieldLabel="Commerce provider"
-                                            name="./cq:catalogDataResourceProviderFactory"
-                                            renderReadOnly="{Boolean}true"
-                                            required="{Boolean}true"/>
+                                        required="{Boolean}true"
+                                        renderRedOnly="{Boolean}true"
```
Comment (herzog31) [inline content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/_cq_dialog/.content.xml]: This should be `renderReadOnly` right?

Comment (laurentiumagureanu) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/_cq_dialog/.content.xml]: yes, will fix it right away

File: content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp (hunk 1 above)
Comment (laurentiumagureanu) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp]: I added a new node here https://github.com/adobe/commerce-cif-connector/blob/4d04e8f1564e68564b22515f327c14639090570b/content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/_cq_dialog/.content.xml#L48

**Review by herzog31 — APPROVED** [review]

File: content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp (hunk 1 above)
Comment (dplaton) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp]: But is that node a new component. If so, why are we using JSPs and not HTML?

Comment (laurentiumagureanu) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp]: I use some Java code to get the path of referenced configuration [here](https://github.com/adobe/commerce-cif-connector/blob/19a2bb3e957fd6dce7f65b0b0a89cec4819e90c4/content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp#L23-L38)
I'm not sure I can do this with HTL

Comment (dplaton) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp]: You can't, you have to use a Sling Model, this is hot it works.

Comment (laurentiumagureanu) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/cifrootfolder/reference/reference.jsp]: @dplaton switched to HTL

**Review by dplaton — APPROVED** [review]

**Review by cjelger — APPROVED** [review]

**Review by herzog31 — APPROVED** [review]

**Pull request merged**